crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -351,7 +351,7 @@ where
             //       on a validation failure?
             // any error, is an error for all
             Err(e) => {
-                if log::log_enabled!(log::Level::Debug) {
+                if tracing::level_enabled!(tracing::Level::DEBUG) {
```

Comment:
this should probably use the [`event_enabled!`](https://docs.rs/tracing/latest/tracing/macro.event_enabled.html) macro instead --- the result will be more accurate, as it will specifically test if the level is enabled _in this module path_
```suggestion
                if tracing::event_enabled!(tracing::Level::DEBUG) {
```

this will require setting the minimum `tracing` version to one that includes this macro in the API

Reply:
I switched to `tracing::enabled`